### PR TITLE
Move plugins to a dedicated directory

### PR DESCRIPTION
**Pull request by MrTimscampi**
**Changes**

This moves all the plugins to a directory specifically for them, away from components.

**Issues**

### CODE REVIEW

**Comment by dkanada** [comment]
At least four of the plugins have other assets that should probably move to the new locations.

**Comment by dkanada** [comment]
The `chromecastHelpers` file might also be better in the new location.

File: src/scripts/site.js [hunk 1]
```diff
@@ -479,21 +479,21 @@ var AppInfo = {};
...
+            'plugins/youtubePlayer/plugin',
+            'plugins/backdropScreensaver/plugin',
+            'plugins/logoScreensaver/plugin'
         ];
 
         if (appHost.supports('remotecontrol')) {
-            list.push('components/sessionPlayer');
+            list.push('plugins/sessionPlayer/plugin');
 
             if (browser.chrome || browser.opera) {
```
Comment (ferferga) [inline src/scripts/site.js:495]: Completely unrelated, but we can also add the new Edge Chromium there I suppose. Do you know if it's under the "Chrome" spectrum or it has its own name?

Comment (MrTimscampi) [reply src/scripts/site.js:495]: I don't think we detect it specifically. I don't have it, so no idea what it'd show up as. Someone who has Windows available should check.

**Comment by dkanada** [comment]
Oh, I thought this one had already been merged. Looks like there are conflicts again.

**Pull request merged**